### PR TITLE
add facade function to query issuable RToken using amounts

### PR DESCRIPTION
**Pull request by tbrent**

### CODE REVIEW

File: contracts/facade/FacadeRead.sol [hunk 1]
```diff
@@ -28,26 +28,46 @@ contract FacadeRead is IFacadeRead {
...
-        BasketRange memory basketsHeld = main.basketHandler().basketsHeldBy(account);
-        uint192 needed = rToken.basketsNeeded();
-
-        int8 decimals = int8(rToken.decimals());
-
-        // return {qRTok} = {BU} * {(1 RToken) qRTok/BU)}
-        if (needed.eq(FIX_ZERO)) return basketsHeld.bottom.shiftl_toUint(decimals);
+        // Get basket ERC20s
+        IBasketHandler bh = main.basketHandler();
+        if (!bh.isReady()) return 0;
```
Comment (akshatmittal) [inline contracts/facade/FacadeRead.sol:55]: Do we need this if we're just quoting here?

File: contracts/facade/FacadeRead.sol [hunk 2]
```diff
@@ -28,26 +28,46 @@ contract FacadeRead is IFacadeRead {
...
+        (address[] memory erc20s, uint256[] memory quantities) = bh.quote(FIX_ONE, CEIL);
 
-        uint192 totalSupply = shiftl_toFix(rToken.totalSupply(), -decimals); // {rTok}
+        // Compute how many baskets we can mint with the collateral amounts
+        uint192 baskets = type(uint192).max;
+        for (uint256 i = 0; i < erc20s.length; ++i) {
+            // {BU} = {tok} / {tok/BU}
+            uint192 inBUs = divuu(amounts[i], quantities[i]); // FLOOR
```
Comment (akshatmittal) [inline contracts/facade/FacadeRead.sol:62]: Given this, should add a comment that the quantities must be in the same order as the erc20s in the basket.

**Review by akshatmittal — CHANGES_REQUESTED** [review]

File: contracts/facade/FacadeRead.sol (hunk 1 above)
Comment (tbrent) [reply contracts/facade/FacadeRead.sol:55]: No we don't...

File: contracts/facade/FacadeRead.sol (hunk 2 above)
Comment (tbrent) [reply contracts/facade/FacadeRead.sol:62]: Good point

**Review by akshatmittal — APPROVED** [review]

**Pull request merged**